TeadsSampleApp/Controllers/InRead/Direct/CollectionView/InReadDirectCollectionViewController.swift

```diff
@@ -141,6 +160,9 @@ extension InReadDirectCollectionViewController: TeadsInReadAdPlacementDelegate {
     }
     
     func adOpportunityTrackerView(trackerView: TeadsAdOpportunityTrackerView) {
-        //not relevant in tableView integration
+        elements.insert(.trackerView(trackerView), at: trackerViewItemNumber)
```

Comment:
Quick question: who retain the trackerView if it wasn't added to the view hierarchy directly? Are we sure that we will not have some of retain cycle here or, the invert, a non existing view in the memory sometimes?

Reply:
nothing retains the `trackerView` until implementation owns it